lib/KDropdownMenu.vue

```diff
@@ -175,19 +175,36 @@
         const popoverIsOpen = popover.clientWidth > 0 && popover.clientHeight > 0;
         // set current element and its siblings
         let focusedElement = document.activeElement;
-        let sibling = focusedElement.nextElementSibling;
-        let prevSibling = focusedElement.previousElementSibling;
+        // let sibling = focusedElement.nextElementSibling;
+        // let prevSibling = focusedElement.previousElementSibling;
+
+        //function to check if the sibling is divider
+        const isDivider = element => {
+          return element && element.classList.contains('is-divider');
+        };
 
         // manage rotating through the options using arrow keys
         // UP arrow: .keyCode is depricated and should used only as a fallback
         if ((event.key == 'ArrowUp' || event.keyCode == 38) && popoverIsOpen) {
           event.preventDefault();
+
+          // Checking if previous sibling is divider and if yes then skip it
+          let prevSibling = focusedElement.previousElementSibling;
+          while (prevSibling && isDivider(prevSibling)) {
+            prevSibling = prevSibling.previousElementSibling;
+          }
           prevSibling
             ? this.$nextTick(() => prevSibling.focus())
             : this.$nextTick(() => lastChild.focus());
           // DOWN arrow
         } else if ((event.key == 'ArrowDown' || event.keyCode == 40) && popoverIsOpen) {
           event.preventDefault();
+
+          //Chekcing if next sibling is divider and skipping it
+          let sibling = focusedElement.nextElementSibling;
+          while (sibling && isDivider(sibling)) {
```

Comment:
We could write two separate methods here, to breakdown a little bit this method. so we can just do something like

```javascript
const sibling = this.getNextFocusableSibling(focusedElement);
const prevSibling = this.getPrevFocusableSibling(focusedElement);
```

Reply:
Hey @AlexVelezLl , I've defined prevSibling Already here
<img width="781" alt="Screenshot 2024-11-11 at 12 21 48 AM" src="https://github.com/user-attachments/assets/44baa489-aef1-4882-9a08-be9df819ae5a">
and the variable sibling Is for next focused element.

Reply:
Yes, my suggestion was more to avoid writing more logic to query the next/previouss element in the same `handleOpenMenuNavigation` method. So lets define two new methods in this component `getNextFocusableSibling` and `getPrevFocusableSibling` where we should do the focusable check and the while sentence.

e.g.
```javascript
methods: {
  ...
  getNextFocusableSibling(element) {
    let sibling = focusedElement.nextElementSibling;
    while (sibling && !isFocusable(sibling)) {
      sibling = sibling.nextElementSibling;
    }
    return sibling;
  }
}
```
and lets refactor the definition of these variables `prevSibling` and `sibling` to use the result of these methods.